NTR: The currency Filter should be able to handle both null and undefined (#1409)
shopware/shopware: changelog/_unreleased/2020-10-15-the-currency-filter-can-now-handle-undefined-values-in-the-administration.md

---
title: The currency filter can now handle undefined values in the administration
issue: NTR
author: Steffen Beisenherz
author_email: s.beisenherz@kellerkinder.de 
author_github: Sironheart
---
# Core

> *  

Reviewer: Funny Boy. You have to fill the changelog, too :smile:

---
title: The currency filter can now handle undefined values in the administration
issue: NTR
author: Steffen Beisenherz
author_email: s.beisenherz@kellerkinder.de 
author_github: Sironheart

> ___

Reviewer: ```suggestion
---
```
Wurd dafür mal gestraft. Sieht man in PHPStorm nicht, weiß ich.

# Administration

> *  The currency filter is now able to handle more data types 

Reviewer: ```suggestion
*  Changed behaviour of currency filter to be able to handle more data types 
```

https://github.com/shopware/platform/blob/master/adr/2020-08-03-Implement-New-Changelog.md
Brauchst keywords
